ext/wmq_queue.c

```diff
@@ -689,10 +689,9 @@ VALUE Queue_get(VALUE self, VALUE hash)
     MQLONG   flag;
     MQLONG   messlen;                /* message length received       */
 
-    MQMD     md = {MQMD_DEFAULT};    /* Message Descriptor            */
+    static const MQMD md_def = {MQMD_DEFAULT};
```

Comment:
`md_def` is never mutated, right? If it was, this wouldn't be thread safe.

Reply:
Correct. The only thing we do with it is copy it into `md`.